cmd/egress/cmd.go

```diff
@@ -112,11 +114,12 @@ are set correctly before execution.
 				Timeout:      config.timeout,
 				Tags:         config.cloudTags,
 				InstanceType: config.instanceType,
+				PlatformType: config.platformType,
 				Proxy:        p,
 			}
 
 			// AWS workflow
-			if !config.gcp {
+			if config.platformType == helpers.PlatformAWS || config.platformType == helpers.PlatformHostedCluster {
```

Comment:
We need to test for the supported value in place of the 'else' here.
In case we are running an unsupported value, it is defaulting to GCP and throws error message unrelated to the actual issue
```
$./osd-network-verifier egress --subnet-id subnet-985b18c4 --security-group-id sg-35c0366c --region us-east-1 --platform test  
please set environment variable GCP_PROJECT_ID to the project ID of the VPC
```

Reply:
Good catch! I'll fix that now